src/platform/user/selectors.js

```diff
@@ -11,9 +12,13 @@ export const isLOA1 = state => selectProfile(state).loa.current === 1;
 export const isMultifactorEnabled = state => selectProfile(state).multifactor;
 export const selectAvailableServices = state => selectProfile(state)?.services;
 export const selectPatientFacilities = state =>
-  selectProfile(state)?.facilities?.filter(
-    f => !f.facilityId.startsWith('742'),
-  ) || null;
+  selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => ({
```

Comment:
This is the main global change. This updates the `isCerner` flag to reflect either the MPI status flag or the hardcoded list in the utilities/cerner folder (if the isCernerLive flag is flipped). This should allow testing in staging and still allow the website team to control when new content goes live and the VAOS team to be in sync with that or automatically pick up new Cerner sites.